interface-definitions/system_option.xml.in

```diff
@@ -37,7 +37,155 @@
+                        <description>\n\n  Multiple CPUs can be specified as a comma-separated list.\n  For example: '1,2,5'</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="cpu"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="rcu-no-cbs">
+                    <properties>
+                      <help>Offload Read-Copy-Update (RCU) callback processing to specified CPUs</help>
+                      <valueHelp>
+                        <format>u32:0-511</format>
+                        <description>CPU core</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>&lt;start-end&gt;</format>
+                        <description>CPU cores range (e.g. 4-7)</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format> </format>
+                        <description>\n\n  Multiple CPUs can be specified as a comma-separated list.\n  For example: '1,2,5'</description>
+                      </valueHelp>
+                      <constraint>
+                        <validator name="cpu"/>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                </children>
+              </node>
+              <node name="memory">
+                <properties>
+                  <help>Memory settings</help>
+                </properties>
+                <children>
+                  <leafNode name="disable-numa-balancing">
+                    <properties>
+                      <help>Disable automatic NUMA memory balancing</help>
+                      <valueless/>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="default-hugepage-size">
+                    <properties>
+                      <help>Set default hugepage size (e.g., 2M, 1G)</help>
+                      <completionHelp>
+                        <list>2M 1G</list>
+                      </completionHelp>
+                      <valueHelp>
+                        <format>2M</format>
+                        <description>2 megabytes</description>
+                      </valueHelp>
+                      <valueHelp>
+                        <format>1G</format>
+                        <description>1 gigabyte</description>
+                      </valueHelp>
+                      <constraint>
+                        <regex>(2M|1G)</regex>
+                      </constraint>
+                    </properties>
+                  </leafNode>
+                  <leafNode name="hugepage-count">
+                    <properties>
+                      <help>Allocate number of hugepages for system use</help>
+                      <valueHelp>
+                        <format>u32</format>
```

Comment:
I am not sure how to implement this in CLI, but I assume we need to allow per-node format as well, otherwise it will lead to a mess in multi-CPU systems. From docs:

```
hugepages=0:1,1:2
```
(allocate 1 hugepage on node0 and 2 hugepages on node1)